activate.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+set -e
+
+dotfiles_dir=$(cd "$(dirname "$0")"; pwd)
+
+ln -s --backup -T ${dotfiles_dir}/vim ~/.vim
```

Comment:
I don't think the `-T` or `--backup` options exists on MacOS, so maybe there's a way we can remove them?